src/pages/saksbehandling/avsluttBehandling/avsluttRevurdering/avsluttRevurdering.module.less

```diff
@@ -0,0 +1,17 @@
+@import '~/styles/variables.less';
+
+.begrunnelseContainer {
+    margin-bottom: @spacing;
+}
+
+.revurderingForhåndsvarsletContainer {
+    margin-bottom: @spacing;
+
+    > :not(:last-child) {
```

Comment:
Ingar tipset meg om `gap` som gir spacing til alle unntatt siste :D

Reply:
Det trengs en del mer CSS for at det skal bli seenede som vi vil det skal. på akkurat dette stedet tjener vi lite på å mekke til bare for å kunne bruke gap.